src/main/java/seedu/ddd/logic/commands/list/ListCommand.java

```diff
@@ -13,6 +13,8 @@
 import static seedu.ddd.logic.parser.CliSyntax.PREFIX_SERVICE;
 import static seedu.ddd.logic.parser.CliSyntax.PREFIX_TAG;
 
+import seedu.ddd.logic.commands.Command;
+
 /**
  * Lists either Contacts or Events in the address book to the user.
  */
```

Comment:
Are we updating `COMMAND_USAGE` in a separate PR? We could either do it here, or create a big PR for standardizing command usage.